webapp/components/wpt-amend-metadata.js

```diff
@@ -164,6 +164,9 @@ class AmendMetadata extends LoadingState(PathInfo(ProductInfo(PolymerElement)))
               <img class="browser" src="[[displayMetadataLogo(node.product)]]">
               :
               <paper-input label="Bug URL" value="{{node.url}}" autofocus></paper-input>
+              <template is="dom-if" if="[[!node.product]]">
```

Comment:
Here, perhaps the bug and label can go on separate lines?

Linking to some documentation about bug and label triage here would be helpful too.

Reply:
I do feel like the UI looks cleaner with `bug` and `label` on the same line but let me add some documentation on the top